packages/odyssey-react/src/components/Banner/Banner.module.scss

```diff
@@ -68,39 +62,51 @@
 
 .infoVariant {
   background-color: var(--InfoBackgroundColor);
-}
 
-.successVariant {
-  background-color: var(--SuccessBackgroundColor);
+  .icon {
+    color: var(--InfoIconColor);
+  }
 }
 
 .cautionVariant {
   background-color: var(--CautionBackgroundColor);
+
+  .icon {
+    color: var(--CautionIconColor);
+  }
 }
 
 .dangerVariant {
   background-color: var(--DangerBackgroundColor);
+
+  .icon {
+    color: var(--DangerIconColor);
+  }
 }
 
-@media screen and (max-width: (calc(var(--MaxLineLength) * 1.5))) {
+@media screen and (max-width: (calc(33rem * 1.5))) {
```

Comment:
@jeffbelser-okta this is a temporary change to ensure our RWD styles work.

Currently, the theme provider doesn't recognize and map `var(--MaxLineLength)` in this context, so Banner has not been behaving responsively.

Reply:
Yup. I had thought so too! I remember troubleshooting a similar issue with you in the past. Maybe we accounted for nesting within `calc()` or `@media` but not both? Sadly, this gets compiled into a plain custom property with no value. 😞 

Happy to assist however I can with the follow-up ticket.